lib/streamlit/runtime/media_file_manager.py

```diff
@@ -179,44 +180,50 @@ def __init__(self):
             str, Dict[str, MediaFile]
         ] = collections.defaultdict(dict)
 
-    def __repr__(self) -> str:
-        return util.repr_(self)
+        self._lock = threading.RLock()
```

Comment:
We could use a `Lock` here - I reached for `RLock` to guard against future modifications accidentally introducing deadlocks. (I'm definitely open to changing it to a Lock!)